porcupine/__main__.py

```diff
@@ -29,6 +32,28 @@
 """
 
 
+def open_files(files: Iterable[str]) -> None:
+    tabmanager = get_tab_manager()
+    for path_string in files:
+        if path_string == "-":
```

Comment:
Presumably this doesn't work when opening from `tk send`. It would be nice if we could fix it (here or in a separate PR), so that you could do

```
$ some long command | porcu -
```

to show the output of the command in the currently running Porcupine instance.

Reply:
Shall we use an ipc file for it, or do you have some other idea?

Reply:
Yep, it doesn't